app/queries/explorer_stake_account_query.rb

```diff
@@ -12,9 +14,16 @@ def initialize(withdrawer: nil, staker: nil, vote_account: nil, stake_pubkey: ni
   def call(page: 1, per: 20)
     explorer_stake_accounts = ExplorerStakeAccount.where(
       network: @network,
-      epoch: EpochWallClock.by_network(@network).first&.epoch
+      epoch: EpochWallClock.by_network(@network).last&.epoch
```

Comment:
Why changed to .last? 
Epochs are in descending order in by_network scope

Also, we should only query .count here, to avoid pulling the collection twice, for two different epochs.